#!/bin/sh /usr/share/dpatch/dpatch-run
## 04-fix-manpage-warning.dpatch
##
## DP: Fix warning in manpage caused by unknown symbol (fixes a lintian
## DP: warning).

diff --git a/doc/cscope.1 b/doc/cscope.1
index 3a6a6e6..c41a59e 100644
--- a/doc/cscope.1
+++ b/doc/cscope.1
@@ -1,4 +1,3 @@
-.PU
 .TH CSCOPE "1" "August 2003" "The Santa Cruz Operation"
 .SH NAME
 cscope - interactively examine a C program 
